mw-webapp/src/component/header/Header.tsx

```diff
@@ -61,6 +64,12 @@ export const Header = () => {
         />
       </a>
       <div className={styles.headerButtonsContainer}>
+        <Tooltip
+          position={PositionTooltip.BOTTOM}
+          content="Theme"
```

Comment:
What do you think about move Tooltip inside themeSwitcher, then you can write more clear content part. For example, "Click to turn on light theme" and "Click to turn on dark theme". What do you think?

Reply:
![image](https://github.com/tritonJS826/masters-way/assets/112805668/3aff365f-c4bc-4ffe-ad5c-4c991ad96319)
![image](https://github.com/tritonJS826/masters-way/assets/112805668/d1e9f9e5-1266-4a9c-b34a-01696dd05a92)
I think it's a good idea, but the line is too long 
now we have
![image](https://github.com/tritonJS826/masters-way/assets/112805668/42f4f27e-efce-4884-ae98-53333a727eb8)

Reply:
As for me the short tooltip(Theme) is better. What do you think?

Reply:
I vote for the middle variant "Switch to dark theme" or "Toggle theme"